src/hotspot/share/jfr/metadata/metadata.xml

```diff
@@ -558,6 +558,11 @@
     <Field type="ulong" contentType="bytes" name="used" label="Used" />
   </Event>
 
+  <Event name="JitRestart" category="Java Virtual Machine, Compiler" label="JIT restart" stackTrace="false" startTime="false" thread="true">
```

Comment:
"JITRestart" . The convention for other events have been to use capital laters for acronyms. 

"JIT restart" -> "JIT Restart"